src/components/TextInput/TextInput.stories.mdx

```diff
@@ -138,6 +141,10 @@ When the `type` prop is set to `"textarea"`, the `<textarea>` element
 is rendered instead of the `<input>` element. This element follows all the same
 accessibility rules described above.
 
+If the `type` prop is set to `"email"`, `"tel"`, or `"url"`, the appropriate
```

Comment:
To support this info, add another link to the `Resources` list:

MDN HTML attribute: autocomplete
https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete